test/integration/query-tests/fill-extrusion/top-down/style.json

```diff
@@ -0,0 +1,141 @@
+{
+  "version": 8,
+  "metadata": {
+    "test": {
+      "debug": true,
+      "width": 200,
+      "height": 400,
+      "queryGeometry": [
+        [0, 0],
+        [200, 400]
+      ]
+    }
+  },
+  "pitch": 0,
```

Comment:
Will this work when pitch is greater than zero?

Reply:
yep, we already have another test that covers for that, the bug mainly manifested at low pitches.